sul.yaml

```
@@ -1,3 +1,7 @@
 alerts:
+  - from: '2020-11-23T12:00:00-00:00' 
+    to: '2020-12-06T12:00:00-12:00'
```

```suggestion
  - from: '2020-11-23T00:00:00-07:00' 
    to: '2020-12-07T00:00:00-07:00'
```

I _think_ this change would say effectively "Will start at the beginning of 11/23 and will stay up until the beginning of 12/7" (using the Pacific time zone offset of -07:00).  Might want to get somebody to double-check me on that though.